pkgs/by-name/db/dbus-cxx/package.nix

```diff
@@ -0,0 +1,40 @@
+{
+  fetchFromGitHub,
+  lib,
+  stdenv,
+  cmake,
+  ninja,
+  pkg-config,
+  libsigcxx30,
+}:
+stdenv.mkDerivation {
+  name = "dbus-cxx";
```

Comment:
You need to have a version in mkDerivation. Move the tag here and rename it then use finalAttrs to recursively access version on the src attr.

Reply:
Okay. Haven't seen this pattern before, is the newest commit what you mean?